vendor/vendor.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
Sorry, I actually missed that file.  The tool used was [govendor](https://github.com/kardianos/govendor).  If you guys have another standard way of bringing in vendor dependencies, let me know and I'll do that instead.